charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -56,11 +32,13 @@ spec:
               fieldPath: metadata.namespace
         - name: SYSFS_ROOT
           value: "/sysfs"
-        # Use this to tell kubelet-plugin where the DRI devices nodes should be.
+        # Only use devdri when using fake devfs with device-faker.
+        # Use this to tell kubelet-plugin where the fake DRI devices nodes are.
         # This will be prefix for CDI devices, runtime will try to mount devices
-        # with this prefix into workloads.
+        # with this prefix into workloads, so the path should be equal to where
+        # fake DRI files are in real host.
         #- name: DEV_DRI_PATH
-        #  value: "/fake/dri"
+        #  value: "/tmp/test-0123456789/dev/dri"
```

Comment:
This is not needed, will only confuse people, let's have it in source code repo deployment files only for now.

Reply:
Ok, I will redo that. 
I used our public repo when creating the chart https://github.com/intel/intel-resource-drivers-for-kubernetes/blob/main/deployments/gpu/resource-driver.yaml to update needed files. That is a bit confusing because it is in public repository and available for end users.

Reply:
I understand the confusion, but you need to understand the purpose. When deploying the yamls manually - you can change that to point to where you've created the fake sysfs and devfs. With helm - we do not expect anyone to change the template, right? Instead you would use templating approach and hide these behind a flag in values.yaml and the proper value that can be set when installing the chart...

Reply:
This will look better later in Q4 when it will be possible to use the fake sysfs generator as an init container...